static/src/javascripts/lib/page.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+
+import $ from 'lib/$';
+import config from 'lib/config';
+import { getBreakpoint } from 'lib/detect';
+
+type yesable = ?(arg: boolean | string) => boolean;
+type noable = ?() => boolean;
+
+const isit = (
+    isTrue: ?(string | boolean),
+    yes: yesable,
+    no: noable,
+    arg: any
+): boolean => {
+    if (isTrue) {
+        return yes ? yes(arg || isTrue) : arg || !!isTrue;
```

Comment:
Are these ternaries necessary, given that `yes` and `no` are not nullable / optional?